scripts/fg_launcher_mac.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#flightgear appimage setup script for flightgear-control project in Mac.
+  #FlightGear-2020.3.17.dmg: Install FlightGear to Desktop or User folder Eg: /Users/<userName>/flightGear/FlightGear.app
+  #FlightGear-2020.3.17-data and FlightGear-2020.3.17-update-data: Using terminal, unzip both the folders
+  #Add path of fgfs (Eg: /Users/<userName>/flightGear/FlightGear.app/Contents/MacOS) to $PATH
+  #FG_BIN_PATH : Set the path of fgfs folder to FG_BIN_PATH
+
+APPIMAGE_FILE=fgfs
+FG_BIN_PATH=`whereis -b $APPIMAGE_FILE | awk '{print $2}'`
+
+#############################
+#explicitly set a display
+
+DISPLAY_STR=${DISPLAY:-":0.0"}
+
+echo "Using display $DISPLAY_STR"
+
+#############################
+
+if [ -z "$FG_BIN_PATH" ]; then
+    echo "Could not find FlightGear AppImage on path."
+    exit 1
+else
+    echo "Found FlightGear AppImage at $FG_BIN_PATH"
+
+    FG_BIN_DIR=`dirname $FG_BIN_PATH  | rev | cut -d/ -f4- | rev`
```

Comment:
Is just using `dirname` not sufficient on mac?

Reply:
dirname is not sufficient. Because path of the app in Mac is /Users/<userName>/flightGear/FlightGear.app/Contents/MacOS/fgfs. If dirname is used, it gives /Users/<userName>/flightGear/FlightGear.app/Contents/MacOS, which is read-only and not a directory